boards/arm/tms570/launchxl-tms57004/scripts/flash-sram.ld

```diff
@@ -43,8 +43,6 @@ MEMORY
   sram (rwx) : ORIGIN = 0x08000000, LENGTH = 32K
 }
 
-OUTPUT_ARCH(arm)
-OUTPUT_FORMAT(elf32-bigarm)
```

Comment:
@xiaoxiang781216  @liuguo09  Notice that ARM big-endian is required for the TMS470 family.  Does this change break TMS570?

Reply:
I tried to build a TMS570 configuration, but it needs an armeb-nuttx-eabi-gcc toolchain and don't have one right now.

Reply:
I suppose the toolchain can select the right endian base on the command line option if the linker script file doesn't specify one explicitly.

Reply:
> 
> 
> I suppose the toolchain can select the right endian base on the command line option if the linker script file doesn't specify one explicitly.

All of these untested changes to code is irresponsible and unprofessional.